### PR TITLE
[EN/ID] Update `History of osu!/2007`

### PR DESCRIPTION
**Pull request by daud-ahn**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by DoryanR** [comment]
I think it would be interesting to include references in links redirecting directly to the bottom of the page, like in the [2014](https://github.com/ppy/osu-wiki/blob/master/wiki/History_of_osu!/2014/en.md) and [2015](https://github.com/ppy/osu-wiki/blob/master/wiki/History_of_osu!/2015/en.md) articles to stay consistent.

**Comment by daud-ahn** [comment]
References found:  
ouentest: https://twitter.com/ppy/status/1141394137637249025?s=20
pUpdater and osume  https://osu.ppy.sh/community/forums/topics/4?n=1

File: wiki/History_of_osu!/2007/id.md [hunk 1]
```diff
@@ -1,16 +1,12 @@
----
-outdated: true
----
-
-# 2007
+# Sejarah osu! 2007
 
-Berikut ini adalah beberapa sejarah osu! sejak awal. Terima kasih khusus kepada [Sinistro](https://osu.ppy.sh/users/5530) telah membantu menyusun konten 2007/2008.
+Berikut ini adalah beberapa peristiwa besar dalam sejarah osu! sejak awal berdirinya. Terima kasih khusus kepada [Sinistro](https://osu.ppy.sh/users/5530) karena telah membantu menyusun konten tahun 2007/2008.
```
Comment (ZeroPyrozen) [inline wiki/History_of_osu!/2007/id.md:3]:
```suggestion
Berikut ini adalah beberapa peristiwa besar dalam sejarah osu! sejak awal berdirinya. Terima kasih khusus kepada [Sinistro](https://osu.ppy.sh/users/5530) karena telah membantu menyusun konten tahun 2007/2008.
```
```suggestion
Berikut ini adalah beberapa peristiwa besar yang terjadi di osu! selama tahun 2007. Terima kasih kepada [Sinistro](https://osu.ppy.sh/users/5530) atas bantuannya dalam menyusun sejarah osu! tahun 2007/2008.
```

File: wiki/History_of_osu!/2007/id.md [hunk 2]
```diff
@@ -1,16 +1,12 @@
...
+osu! dimulai sebagai *proof-of-concept* bernama "ouentest"[[1]][r]. Versi pertama dirilis pada 1 Juli 2007 untuk sekelompok kecil teman dekat [peppy](https://osu.ppy.sh/users/2) dan memiliki sebuah [editor beatmap](/wiki/Beatmap_Editor) dan mode permainan yang sangat mendasar. Editor hanya dapat membuat [beatmap](/wiki/Beatmap) yang *sangat* sederhana tanpa fungsi *seeking* atau [*snapping*](/wiki/Beatmapping/Snapping) dan tidak ada slider atau spinner, hanya circle dan penghitung skor/kombo yang mendasar. Rilis pertama memakan waktu sekitar 16 jam waktu coding dan dilakukan dalam *single all-nighter*. `pUpdater` diperkenalkan (lekas dikenal sebagai `osume`)[[2]][r] untuk menangani pembaruan ke versi terbaru.
```
Comment (ZeroPyrozen) [inline wiki/History_of_osu!/2007/id.md:9]:
```suggestion
osu! dimulai sebagai *proof-of-concept* bernama "ouentest"[[1]][r]. Versi pertama dirilis pada 1 Juli 2007 untuk sekelompok kecil teman dekat [peppy](https://osu.ppy.sh/users/2) dan memiliki sebuah [editor beatmap](/wiki/Beatmap_Editor) dan mode permainan yang sangat mendasar. Editor hanya dapat membuat [beatmap](/wiki/Beatmap) yang *sangat* sederhana tanpa fungsi *seeking* atau [*snapping*](/wiki/Beatmapping/Snapping) dan tidak ada slider atau spinner, hanya circle dan penghitung skor/kombo yang mendasar. Rilis pertama memakan waktu sekitar 16 jam waktu coding dan dilakukan dalam *single all-nighter*. `pUpdater` diperkenalkan (lekas dikenal sebagai `osume`)[[2]][r] untuk menangani pembaruan ke versi terbaru.
```
```suggestion
osu! dimulai sebagai *proof-of-concept* bernama "ouentest"[[1]][r]. Versi pertama dirilis pada tanggal 1 Juli 2007 untuk sekelompok kecil teman dekat [peppy](https://osu.ppy.sh/users/2) dan memiliki sebuah [editor beatmap](/wiki/Beatmap_Editor) dan mode permainan yang sangat mendasar. Editor hanya dapat membuat [beatmap](/wiki/Beatmap) yang *sangat* sederhana tanpa fungsi *seeking* atau [*snapping*](/wiki/Beatmapping/Snapping) dan tidak ada slider atau spinner, hanya circle dan penghitung skor/kombo yang mendasar. Rilis pertama memakan waktu sekitar 16 jam waktu coding dan dilakukan dalam *single all-nighter*. `pUpdater` diperkenalkan (lekas dikenal sebagai `osume`)[[2]][r] untuk menangani pembaruan ke versi terbaru.
```

File: wiki/History_of_osu!/2007/id.md [hunk 3]
```diff
@@ -20,25 +16,25 @@ osu! dimulai dengan nama konsep awalnya bernama "ouentest". Versi pertama dirili
...
 
-Menu utama telah diperbarui dengan tampilan baru, dan editor menerima banyak fungsi baru, termasuk dukungan salin dan tempel (copy dan paste). Bilah dan kursor menu Windows ditambahkan, membuat fitur editor jauh lebih mudah diakses dibanding sebelumnya. Slider lebih banyak mendapat dukungan, dengan ticks datang dan menghilang saat bola slider melewati mereka. Penumpukan hit circle (Stacking) telah hadir. Skor terbaik lokal jauh lebih didukung. Format file `osu!` direvisi dan dirubah sepenuhnya, tetapi telah mencakup konversi file lama ke format baru.
+Menu utama telah diperbarui dengan tampilan baru, dan editor menerima banyak fungsi baru, termasuk dukungan salin dan tempel (copy dan paste). Menu tab dan menu konteks (*context menu*) ditambahkan, membuat fitur editor jauh lebih mudah diakses dibanding sebelumnya; slider lebih banyak mendapat dukungan, dengan ditambahkannya [slider tick](/wiki/Beatmapping/Slider_tick); diperkenalkannya [stacking](/wiki/Mapping_Techniques/Stack) [hit circles](/wiki/Hit_object/Hit_circle); dukungan lebih baik untuk skor terbaik lokal; dan format file `osu!` direvisi dan dirubah sepenuhnya, tetapi telah mencakup konversi file lama ke format baru.
```
Comment (ZeroPyrozen) [inline wiki/History_of_osu!/2007/id.md:31]:
```suggestion
Menu utama telah diperbarui dengan tampilan baru, dan editor menerima banyak fungsi baru, termasuk dukungan salin dan tempel (copy dan paste). Menu tab dan menu konteks (*context menu*) ditambahkan, membuat fitur editor jauh lebih mudah diakses dibanding sebelumnya; slider lebih banyak mendapat dukungan, dengan ditambahkannya [slider tick](/wiki/Beatmapping/Slider_tick); diperkenalkannya [stacking](/wiki/Mapping_Techniques/Stack) [hit circles](/wiki/Hit_object/Hit_circle); dukungan lebih baik untuk skor terbaik lokal; dan format file `osu!` direvisi dan dirubah sepenuhnya, tetapi telah mencakup konversi file lama ke format baru.
```
```suggestion
Menu utama telah diperbarui dengan tampilan baru, dan editor menerima banyak fungsi baru, termasuk dukungan salin dan tempel (copy dan paste). Menu tab dan menu konteks (*context menu*) ditambahkan, membuat fitur editor jauh lebih mudah diakses dibanding sebelumnya; slider lebih banyak mendapat dukungan, dengan ditambahkannya [slider tick](/wiki/Beatmapping/Slider_tick); diperkenalkannya [stacking](/wiki/Mapping_Techniques/Stack) [hit circles](/wiki/Hit_object/Hit_circle); dukungan lebih baik untuk skor terbaik lokal; dan format berkas `osu!` direvisi dan dirubah sepenuhnya, tetapi telah mencakup konversi berkas lama ke format baru.
```

**Review by ZeroPyrozen — CHANGES_REQUESTED** [review]

**Review by ZeroPyrozen — APPROVED** [review]
Looks good to me 👍

File: wiki/History_of_osu!/2007/en.md [hunk 4]
```diff
@@ -44,7 +44,7 @@ Gameplay was enhanced with the debut of [combo fire](/wiki/Glossary/Combo_fire),
 
 ![](img/2007-09.jpg)
 
-September 17, 2007 was the "birth" of osu! as far as the public was concerned. The forums as we know them today were established, and hosted from peppy's home domain (`ppy.sh`). Attention was gained via a news announcement on [bemanistyle](http://bemanistyle.com) which drew quite a few interested people with previous rhythm game experience.
+September 17, 2007 was the "birth" of osu! as far as the public was concerned. The forums as we know them today were established, and hosted from peppy's home domain (`ppy.sh`). Attention was gained via a news announcement on [bemanistyle](http://bemanistyle.com) in [this post](https://web.archive.org/web/20071221020801/http://www.bemanistyle.com/index.php?news_id=483) which drew quite a few interested people with previous rhythm game experience.
```
Comment (Walavouchey) [inline wiki/History_of_osu!/2007/en.md:47]:
```suggestion
September 17, 2007 was the "birth" of osu! as far as the public was concerned. The forums as we know them today were established, and hosted from peppy's home domain (`ppy.sh`). Attention was gained via a [news announcement](https://web.archive.org/web/20071221020801/http://www.bemanistyle.com/index.php?news_id=483) on [bemanistyle](http://bemanistyle.com) which drew quite a few interested people with previous rhythm game experience.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

**Comment by Walavouchey** [comment]
@daud-ahn are you going to apply the review or should we make the edits on the translation ourselves? just a small style issue, but this is good to go otherwise (cc @ZeroPyrozen)

**Comment by ZeroPyrozen** [comment]
yes, it's good enough. Change line 47 of `id.md` to `osu! mulai mendapat perhatian saat disorot melalui [pengumuman berita](https://web.archive.org/web/20071221020801/http://www.bemanistyle.com/index.php?news_id=483) di forum [bemanistyle](http://www.bemanistyle.com), yang menarik beberapa orang yang memiliki pengalaman game ritme sebelumnya.` if change above applied

File: wiki/History_of_osu!/2007/id.md [hunk 5]
```diff
@@ -48,7 +44,7 @@ Peningkatan gameplay lebih baik dengan hadirnya combo fire, bar HP, mouse trail,
 
 ![](img/2007-09.jpg)
 
-2007-09-17 adalah tanggal kelahiran publik osu!. Forum-forum yang kita kenal sekarang dibentuk pada saat itu, dan dihosting menggunakan domain rumahan peppy (`ppy.sh`). osu! mulai mendapat perhatian saat disorot melalui pengumuman berita di [bemanistyle](http://www.bemanistyle.com), menarik beberapa orang yang memiliki pengalaman game ritme sebelumnya.
+Tanggal 17 September 2007 merupakan tanggal peluncuran osu! secara publik. Forum-forum yang kita kenal sekarang dibentuk pada saat itu dan dihosting menggunakan domain rumahan peppy (`ppy.sh`). osu! mulai mendapat perhatian saat disorot melalui pengumuman berita di [bemanistyle](http://www.bemanistyle.com) dalam [postingan ini](https://web.archive.org/web/20071221020801/http://www.bemanistyle.com/index.php?news_id=483), yang menarik beberapa orang yang memiliki pengalaman game ritme sebelumnya.
```
Comment (Walavouchey) [inline wiki/History_of_osu!/2007/id.md:47]:
```suggestion
Tanggal 17 September 2007 merupakan tanggal peluncuran osu! secara publik. Forum-forum yang kita kenal sekarang dibentuk pada saat itu dan dihosting menggunakan domain rumahan peppy (`ppy.sh`). osu! mulai mendapat perhatian saat disorot melalui [pengumuman berita](https://web.archive.org/web/20071221020801/http://www.bemanistyle.com/index.php?news_id=483) di forum [bemanistyle](http://www.bemanistyle.com), yang menarik beberapa orang yang memiliki pengalaman game ritme sebelumnya.
```

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**